ui/client/actions/nk/nodeDetails.ts

```diff
@@ -0,0 +1,33 @@
+import {ThunkAction} from "../reduxTypes"
+import HttpService from "../../http/HttpService"
+import _ from "lodash"
+
+export type NodeDataUpdated = { type: "NodeDataUpdated", nodeData: any}
+export type NodeValidationUpdated = { type: "NODE_VALIDATION_UPDATED", validationData: any}
+export type NodeDetailsActions = NodeDataUpdated | NodeValidationUpdated
+
+export type NodeValidationData = {
+    parameters? : Map<string, ValidationContext>,
```

Comment:
Map? for sure? maybe it's just Object then you need Record.

Reply:
true, fixed :+1: